std_msgs/Header header

uint8 ROTATION=0
uint8 POSITION=1
uint8 mode

int16 speed
int32 angle

uint8 RETURN_TO_START_POS=2
uint8 STOP_ON_ERROR=3
uint8 RETURN_TO_NULL_POS=4
uint8 NO_ERROR = 5
uint8 error_flag

float64 maxload